conf/application.conf

```diff
@@ -113,6 +113,11 @@ feature-switch {
   passDeleteIntentHeader.enabled = true
   desIf_Migration.enabled = false
 
+  additional_directorship_dividend_data {
```

Comment:
Please hyphenate and just stop at dividend i.e. `additional-directorship-dividend`